lib/allocatedWorkers.spec.ts

```diff
@@ -205,4 +207,27 @@ describe('allocatedWorkersAPI', () => {
       expect(data).toEqual({ allocations: ['foo'] });
     });
   });
+
+  describe('addWorkerAllocation', () => {
+    it('should work properly', async () => {
```

Comment:
Its not a massive issue, but you might want to make the name of this test more meaningful than "it works".